projects/sigstore/build.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -eu
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+fuzzer=fuzz_loadCertificates.a
+cd "$GOPATH"/src/github.com/sigstore/sigstore/test/fuzz/pem
+
+go-fuzz -tags go-fuzz -func FuzzLoadCertificates -o $fuzzer  .
+
+$CXX $CXXFLAGS $LIB_FUZZING_ENGINE $fuzzer -o $OUT/$fuzzer
```

Comment:
nit: you can just use the `compile_go_fuzzer` script instead (https://google.github.io/oss-fuzz/getting-started/new-project-guide/go-lang/#buildsh)

Reply:
I am having trouble with compiling with it. TBH, I don't mind if you can fix it and push it. I can use the same for sigstore/sigstore ClusterFuzzLite.

Reply:
I've fixed up your PR to use `compile_go_fuzzer`.

Reply:
Thank you!